### PR TITLE
set aws credentials before pushing to mirror repository

### PR DESCRIPTION
**Pull request by federico-plutoflume**
Set aws env vars before pushing to a mirror repository

_note: there are a lot of other changes because we never committed them and pushed only to the docker registry_

### CODE REVIEW

File: assets/common.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ start_docker() {
   declare -fx try_start
   trap stop_docker EXIT
 
-  if ! timeout -t ${STARTUP_TIMEOUT} bash -ce 'while true; do try_start && break; done'; then
+  if ! timeout ${STARTUP_TIMEOUT} bash -ce 'while true; do try_start && break; done'; then
```
Comment (federico-plutoflume) [inline assets/common.sh:96]: this option changed in latest versions of thebase image

Comment (alex-plutoflume) [reply assets/common.sh:96]: This is fixed in the upstream repo. Do you think we should merge in the upstream master branch?

Comment (federico-plutoflume) [reply assets/common.sh:96]: yes, I'll give it a try in this PR. if it's too messy I'll open a new one :+1:

Comment (federico-plutoflume) [reply assets/common.sh:96]: @alex-plutoflume I'm testing this with a couple of pipelines

Comment (alex-plutoflume) [reply assets/common.sh:96]: cool, let me know once you're happy with it and I'll approve

**Pull request merged**